server/sonar-web/src/main/js/apps/issues/workspace-list-view.js

```diff
@@ -136,6 +141,16 @@ export default WorkspaceListView.extend({
   destroyChildren () {
     WorkspaceListView.prototype.destroyChildren.apply(this, arguments);
     this.$('.issues-workspace-list-component').remove();
+  },
+
+  serializeData () {
+    /* eslint-disable no-console */
+    console.log(this.options.app.state.toJSON());
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS2228)